src/ApplePayGateway.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+/**
+ * Vindicia ApplePay Gateway
+ * This Apple Pay Gateway provides the functionality to authorize, complete authorize and capture
+ * Apple Pay payments for Apple Pay on the web.
+ * 
+ * // TODO: Add documentation about functionality for capture and completeAuthorize methods to capture
+ * // and authorize payments.
+ * 
+ * Apple Pay is available on all iOS devices with a Secure Element—an industry-standard, certified
+ * chip designed to store payment information safely. On macOS, users must have anApple Pay-capable
+ * iPhone or Apple Watch to authorize the payment, or a MacBook Pro
+ * with Touch ID.
+ *
+ * You use the same username and password as you do with the regular Vindicia gateway.
+ *
+ * Optional parameters set by this gateway:
```

Comment:
are these really optional? i thought you needed the cert

Reply:
Well the cert password certainly is.

Technically, I split the cert into a key and a pem but I believe there may be a way to only use one cert. So, I put that they were optional since you can send the request with or without a cert.

Reply:
if you send the request without a cert, what happens? does apple accept that? how do they know it's real?

Reply:
Apple sends an error. I can reword it, I wanted the user to know that they were optional incase they had a different cert setup.

Reply:
This library doesn't support a different cert setup though, right? It may be optional for Apple Pay, but not if you're using this library?

Reply:
There's definitely more than one cert setup for Apple Pay though. The way I did it to get it to work is not listed on Apple's website.

Reply:
If you were a user of this library, how would you make it work with a different cert setup? You're writing documentation for this omnipay library, not apple pay, and it looks to me like this library needs the two certs in order to successfully make an apple pay transaction.

If I'm missing something and there is a different set of parameters that you could specify to make a successful apple pay transaction, can you let me know what those are?

Reply:
It's really just pem cert and an additional key cert vs a pem cert that contains a key cert in it as well.

Reply:
I never got it to work, that is the way apple intended it though.

Reply:
sounds good! the docs just need to say one way to use it, it's fine if we don't provide every possible way to use it